packages/commonwealth/server/passport/magic.ts

```diff
@@ -315,56 +325,17 @@ async function loginExistingMagicUser({
         },
         { transaction },
       );
-      log.info(
-        `Created SsoToken for invalid state user ${existingUserInstance.id}`,
-      );
-    }
 
-    // remove ghost addresses once replacements have been generated
-    const ghostAddresses =
-      existingUserInstance?.Addresses?.filter(
-        ({ ghost_address }: AddressAttributes) => !!ghost_address,
-      ) || [];
-    for (const ghost of ghostAddresses as AddressAttributes[]) {
-      const replacementAddress = addressInstances.find(
-        ({ community_id, ghost_address }: AddressAttributes) =>
-          !ghost_address && community_id === ghost.community_id,
+      // TODO: Check if ONLY after first token created for a canonical address?
```

Comment:
still in progress?

Reply:
this is ready for review... I merged your fix and it worked for me locally